Cython/Compiler/ModuleNode.py

```diff
@@ -1553,6 +1554,27 @@ def generate_new_function(self, scope, code, cclass_entry):
         code.putln(
             "}")
 
+    def generate_del_function(self, scope, code):
+        tp_slot = TypeSlots.ConstructorSlot("tp_finalize", '__del__')
+        slot_func = scope.mangle_internal("tp_finalize")
```

Comment:
It's enough to do this once.
```suggestion
        slot_func_cname = scope.mangle_internal("tp_finalize")
```